src/__tests__/__snapshots__/macro.ts.snap

```diff
@@ -1,8 +1,8 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`macros as function: as function 1`] = `
+exports[`preval as function: as function 1`] = `
```

Comment:
I didn't notice this change when making the PR because we had a stage of "unknown plugin" in the middle.

Is this an important difference to fix? I think it just needs to change the `pliginName` in the test file?

Reply:
I'm not concerned. I think `preval` makes more sense anyway.